lisa/tools/powershell.py

```diff
@@ -23,8 +23,13 @@ def run_cmdlet(
         force_run: bool = False,
         sudo: bool = False,
         fail_on_error: bool = True,
+        timeout: int = 600,
     ) -> str:
-        result = self.run(cmdlet, force_run=force_run, sudo=sudo)
+        # cmdlet can have `"` characters, so we need to escape them
+        cmdlet = cmdlet.replace('"', '\\"')
```

Comment:
Does the `\` work in PowerShell? Should it be "`"?

Reply:
Something like `powershell "echo \"hello\""` worked fine for me in command prompt

Reply:
The Powershell escape is complex, it depends on what results it needs. We can see if there is error, how to fix it in future. the `\"` just remove the quote, it may not be what needs.

```powershell
powershell "echo \"hello\""
```
returns
```cmd
hello
```

```powershell
powershell "echo hel`\"lo"
```
returns
```cmd
hel"lo
```